lib/config/config.go

```diff
@@ -418,6 +420,12 @@ nextPendingDevice:
 	}
 	if cfg.Options.UnackedNotificationIDs == nil {
 		cfg.Options.UnackedNotificationIDs = []string{}
+	} else if cfg.GUI.User != "" && cfg.GUI.Password != "" {
+		for i, key := range cfg.Options.UnackedNotificationIDs {
+			if key == "authenticationUserAndPassword" {
+				cfg.Options.UnackedNotificationIDs = append(cfg.Options.UnackedNotificationIDs[:i], cfg.Options.UnackedNotificationIDs[i+1:]...)
```

Comment:
add a break?
Alternatively, you might need to decrement i

Reply:
Fixed, thank you